packages/runtime/container-runtime/src/opLifecycle/definitions.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IBatchMessage } from "@fluidframework/container-definitions";
+import { CompressionAlgorithms, ContainerRuntimeMessage, ICompressionRuntimeOptions } from "..";
+
+export interface IOutboxOptions {
+    readonly compressionOptions?: ICompressionRuntimeOptions;
+    readonly enableOpReentryCheck?: boolean;
+    readonly maxBatchSizeInBytes: number;
+};
+
+export interface IBatchProcessor {
+    processOutgoing(batch: IBatch): IBatch;
+}
+
+export interface IBatchProcessors {
+    readonly compressor: IBatchProcessor;
+}
```

Comment:
Similar here, can you please add some English narrative on the purpose of interfaces and members?
While I can guess why it's the way it is, it's hard for uneducated reader to understand why we need IBatchProcessors when it looks almost like IBatchProcessor

Reply:
I will rename this.